docs/content/interacting/consistency.mdx

```diff
@@ -55,14 +55,16 @@ if (date_modified + cache_time_to_live_period > Date.now()) {
+| check-cache-limit                | Configures the number of items that will be kept in the in-memory cache used to resolve Check queries                                                                                                                                                                                                                                                             |
+| check-query-cache-enabled        | Enables in-memory caching of Check subproblems. For example, if you have a relation `define viewer: owner or editor`, and the query is `Check(user:anne, viewer, doc:1)`, we'll evaluate the `owner` relation and the `editor` relation and cache both results: `(user:anne, viewer, doc:1) -> allowed=true` and `(user:anne, owner, doc:1) -> allowed=true`.     |
+| check-query-cache-ttl            | Specifies the time that items will be kept in the cache of Check subproblems                                                                                                                                                                                                                                                                                      |
+| check-iterator-cache-enabled     | Enables in-memory caching of database iterators. Each iterator is the result of a database query, for example, usersets related to a specific object, or objects related to a specific user, up to a certain number of tuples per iterator                                                                                                                        |
+| check-iterator-cache-max-results | Configures the number of tuples that will be stored for each database iterator                                                                                                                                                                                                                                                                                    |
+| check-iterator-cache-ttl         | Specifies the time that items will be kept in the cache of database iterators                                                                                                                                                                                                                                                                                     |
+| cache-controller-enabled         | When enabled, cache controller will verify whether check subproblem cache and check iterator cache needs to be invalidated when there is a check or list objects API request. The invalidation determination is based on whether there are recent write or deletes for the store. This feature allows a larger check-query-cache-ttl and check-iterator-cache-ttl.|
```

Comment:
> The invalidation determination is based on whether there are recent write or deletes for the store.
Just if there is recent writes/deletes? Is it not also looking at if the recent writes/deletes are related to the check/LO request?

Reply:
> This feature allows a larger check-query-cache-ttl and check-iterator-cache-ttl.

at the expense of...? 

Thinking of it from a user perspective, it's not clear why this feature is not ON by default.

Reply:
> is it not also looking at if the recent writes/deletes are related to the check/LO request

Partially - if there are many recent changes, it will invalidate all subproblem cache and iterator cache.  If there are only a few recent changes, it will invalidate subproblem cache and only the iterator cache associated with the recent changes.  

I am a bit hesitant to add this distinction here to avoid confusing the reader.

Reply:
> at the expense of
at the expense of additional datastore queries for recent writes and deletes.